pkg/restic/exec_commands_test.go

```diff
@@ -74,12 +74,14 @@ func Test_getLastLine(t *testing.T) {
 second line
 third line
 `, "third line"},
+		{"", ""},
```

Comment:
suggest changing the `tests` struct to 
```
	tests := []struct {
		output []byte
		want   string
	}{
		{[]byte(`last line
`), "last line"},
		{[]byte(`first line
second line
third line
`), "third line"},
		{[]byte(""), ""},
		{nil, ""},
	}
```
and remove change on L#84

Reply:
I tried this originally, but the compiler whined:
```
cannot use nil as type string in field value
```
Apparently you can't assign `nil` to a `string`, so I added another assert rather than convert all the other test cases to `[]byte`. I can change them all if you'd prefer.

Reply:
Yes, that's why I suggested changing the struct also.